src/renderer/transport.ts

```diff
@@ -0,0 +1,16 @@
+import { createTransport } from '@sentry/core';
+import { BaseTransportOptions, Transport, TransportMakeRequestResponse, TransportRequest } from '@sentry/types';
+
+import { getIPC } from './ipc';
+
+/**
+ * Creates a Transport that passes envelopes to the Electron main process.
+ */
+export function makeRendererTransport(options: BaseTransportOptions): Transport {
+  const ipc = getIPC();
+
+  return createTransport(options, async (request: TransportRequest): Promise<TransportMakeRequestResponse> => {
+    ipc.sendEnvelope(request.body);
+    return { statusCode: 200 };
```

Comment:
Should we add a comment here stating why it is fine to always return 200 here? I am guessing because it's just transferring the event over to the main process which will then take care of rate limiting itself.